README.md

```diff
@@ -1,27 +1,26 @@
-# Semantic Versioning Specification
+# Explicit Versioning Specification
 
 To know why I am not using [SemVer]((http://semver.org/)) please see [here](WHY.md).
 
 
 ## Specification
 
-My Semantic Version schema can be represented like `PUBLIC_API.BACKWARDS_INCOMPATIBLE.CODE_IMPROVMENT.BUG_FIX`.
+My Explicit Version schema can be represented like `Release.Incompatible.Compatible.Fix`.
 
 Given `1.0.0.0` as the first version for a **_Production Release_**:
 
-* **BUG_FIX**:
-    + _Security Fix(es)_ will increment version to `1.0.0.1`.
-    + _Bug Fix(es)_ will increment version to `1.0.0.1`.
-* **CODE_IMPROVMENT**:
-    + _New Features_ will increment version to `1.0.1.0`.
-    + _Refracting Code_ that do not impact _Public Api_ will increment version to `1.0.1.0`.
-    + _Deprecating Code_ that do not impact _Public Api_ will increment version to `1.0.1.0`.
-    + _Removing Code_ that do not impact _Public Api_ will increment version to `1.0.1.0`.
-* **BACKWARDS_INCOMPATIBLE**:
-    + _Any Breaking Code Change_ will increment version to `1.1.0.0`.
-*  **PUBLIC_API**:
-    + _Major Code Overhaul_ that impacts a significant number of end points in the _Public Api_ will increment
-        version to `2.0.0.0`.
+* **Release**
```

Comment:
The most left number must be incremented to `1` only when your Software is ready to be used in Production.

Explicit Versioning specification or any other Versioning specification can't enforce that Developers shouldn't use a Developer Release in their Production Software, but Package Manager can/should enforce it.

Resuming, enforcing not using Development Releases in Production is out of scope for any Versioning Specification schema, it can be only recommended as a best practice.